client.cpp

```diff
@@ -1,21 +1,17 @@
-#include <sys/socket.h>
-#include <arpa/inet.h>
-#include <string.h>
-#include <unistd.h>
-#include <stdio.h>
+#include "server.hpp"
```

Comment:
Why are we using `server.hpp` to include? I believe we should have `client.cpp` independent of server.

Reply:
Too many spaces as well, follow the code style.